migrations/1_initialize_schema.up.sql

```diff
@@ -1691,4 +1692,44 @@ CREATE TRIGGER before_item_delete
   FOR EACH ROW
 EXECUTE PROCEDURE decrease_statistics();
 
+CREATE OR REPLACE FUNCTION delete_execution_statistics()
+    RETURNS TRIGGER AS
+$$
+  DECLARE
+  curr_item_id BIGINT;
+  curr_parent_id BIGINT;
+BEGIN
+  IF old.has_children != new.has_children
+  THEN
+    curr_item_id := old.item_id;
+    curr_parent_id := old.parent_id;
+    WHILE curr_item_id IS NOT NULL
+      LOOP
+        UPDATE statistics
+        SET s_counter = 0
+        FROM statistics_field
+        WHERE statistics_field.sf_id = statistics.statistics_field_id
+          AND item_id = curr_item_id
+          AND statistics_field.name LIKE 'statistics$executions$%';
+        curr_item_id := curr_parent_id;
+        curr_parent_id := (SELECT parent_id FROM test_item WHERE item_id = curr_item_id);
+      END LOOP;
+    UPDATE statistics
```

Comment:
why we clean the whole launch statistics if has_children flag has changed ?

Reply:
All parents statistics should be recalculated, including launch, in case some item became parent for another. It can contain defects and executions statistics that should be removed and recalculated based on children

Reply:
let's imaging two suites 
one contains some children with defects and execution statistics
another doesn't contain children
the second gets new children and it's status changes 

all the launch statistics is cleaned up?